reference/4.0/Microsoft.PowerShell.Core/About/about_Line_Editing.md

```diff
@@ -8,48 +8,73 @@ title:  about_Line_Editing
 
 # About Line Editing
 
-## SHORT DESCRIPTION
+## Short description
 
-Describes how to edit commands at the Windows PowerShell command prompt.
+Describes how to edit commands at the PowerShell command prompt.
 
-## LONG DESCRIPTION
+## Long description
 
-The Windows PowerShell console has some useful features to help
-you to edit commands at the Windows PowerShell command prompt.
+The PowerShell console has some useful keyboard shortcuts to help you edit
+commands at the PowerShell command prompt.
 
-### Move Left and Right
+### Add a line
```

Comment:
This does not apply to the version of PSReadLine that shipped in v4.

Reply:
Fixed.